middleware/faq.rst

```diff
@@ -85,46 +79,73 @@ For all but the smallest repos, it's best to start with some kind of guess at wh
 Note that some collections appear multiple times here - once as a top-level collection, and again later as some child of a `~CollectionType.CHAINED` collection (that's what the indentation means here).
 In the future we may be able to remove some of this duplication.
 
-queryDatasetTypes
------------------
+Similar functionality can be accessed from Python using `Butler.collections.query_info <lsst.daf.butler.ButlerCollections.query_info>`.
+
+Dataset Types
+-------------
 
 `Registry.queryDatasetTypes` reports the :ref:`dataset types <daf_butler_dataset_types>` that have been registered with a data repository, even if there aren't any datasets of that type actually present.
 That makes it less useful for exploring a data repository generically, but it's an important tool when you know the name of the dataset type already and want to see how it's defined.
 
-queryDimensionRecords
----------------------
+Dimension Records
+-----------------
 
-`Registry.queryDimensionRecords` is the best way to inspect the metadata records associated with data ID keys (:ref:`"dimensions" <lsst.daf.butler-dimensions_overview>`), and is usually the right tool for those looking for something similar to Gen2's `~lsst.daf.persistence.Butler.queryMetadata`.
+`Butler.query_dimension_records` is the best way to inspect the metadata records associated with data ID keys (:ref:`"dimensions" <lsst.daf.butler-dimensions_overview>`), and is usually the right tool for those looking for something similar to Gen2's `~lsst.daf.persistence.Butler.queryMetadata`.
```

Comment:
It's probably time to stop referring to Gen 2 since no-one reading this is going to know what that is any more.